dir.traversal.targets

```diff
@@ -47,6 +53,13 @@
       <Project Remove="@(Project)" />
       <Project Include="@(ProjectsToBuild)" />
     </ItemGroup>
+
+    <ItemGroup Condition="'$(FilterToTargetGroup)'!=''">
+      <FilteredProjectsToBuild Include="@(Project)" Condition="'%(Project.TargetGroup)'=='$(FilterToTargetGroup)' Or '%(Project.Extension)' == '.proj' Or '%(Project.Extension)' == '.builds'" />
```

Comment:
Can you change this to something similar to what we are doing with OSGroup in PR https://github.com/dotnet/corefx/pull/7535/files#diff-cd0fc5e0bad8102e1a45aa7575bdd102R53.